src/SdkGenerator/SdkGenerator.csproj

```diff
@@ -1,74 +1,85 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-    <PropertyGroup>
-        <OutputType>Exe</OutputType>
-        <TargetFramework>net7.0</TargetFramework>
-        <PackAsTool>true</PackAsTool>
-        <ToolCommandName>SdkGenerator</ToolCommandName>
-        <PackageOutputPath>../../</PackageOutputPath>
-    </PropertyGroup>
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
+    <PackAsTool>true</PackAsTool>
+    <ToolCommandName>SdkGenerator</ToolCommandName>
+    <PackageReadmeFile>docs/README.md</PackageReadmeFile>
```

Comment:
This tells the package where the README is, but the trick is that the path here is relative to the contents of the package, so the path needs to be in sync here and below where we add the README file to the build.